packages/core/src/components/ChoiceList/Choice.jsx

```diff
@@ -19,6 +19,12 @@ export class Choice extends React.PureComponent {
   constructor(props) {
     super(props);
 
+    if (props['inputPlacement'] === 'right') {
+      console.error(
+        `[Deprecated]: 'inputPlacement' is no longer supported and will be removed in a future release, please remove right-to-left <Choice>`
```

Comment:
I'm wondering if we should also update the text in the prop table so folks know it's deprecated.
https://github.com/CMSgov/design-system/pull/508/files#diff-7c1a7d4da2bdf438dffdc0cd5254e7edR205

I'm also wondering if in the future we remove the `inputPlacement` prop altogether

Reply:
yeah I can add it to the prop table too. Yes, we should remove the prop in the future for sure